### ISSUE TITLE
Musicbot disturbed audio

### ISSUE THREAD

**Issue by xRedFox** [TeaSpeak/TeaSpeak#162]
## System Information
***Operating System***: ArchLinux
***Operating System Type*** (x86|x64): x64
***TeaSpeak Version***: 1.1.30 beta 3

## Issue
***Reason for reporting this issue***:
Enhancement

***Detailed Description***:
Every time the music bot is playing a track the audio seems disturbed.

I think this is the problem:
```
[12:49:34] [ERROR] 161 | Found delay while ticking server. (Delay: 516ms)
[12:49:34] [ERROR] 162 | Found delay while ticking server. (Delay: 517ms)
[12:49:34] [ERROR] 163 | Found delay while ticking server. (Delay: 517ms)
[12:49:34] [ERROR] 165 | Found delay while ticking server. (Delay: 515ms)
```
The server is not overloaded... Currently there are 7 people and 31 instances online

No **Logs**, no party: [log_2018-06-03_11_30_28_general.zip](https://github.com/TeaSpeak/TeaSpeak/files/2065877/log_2018-06-03_11_30_28_general.zip)

**EDIT**: I use _contabo_ as host and my plan is _VPS M_ with two intel cpu cores, 6 GB RAM, 500 GB disk space (SSD-boosted), unlimited traffic, 100 mbit/s, ddos protection and ArchLinux x64 as in description.
In other words, it's not a very slow server...
![fuck](https://user-images.githubusercontent.com/19707386/40886000-b9b3bba0-6730-11e8-9733-3660c5a4d287.PNG)

**Comment by WolverinDEV** [comment]
Hey,
i got that what you mean.
Its not total disturbed but sometimes laggy.
I'll implement something like a "buffer" so if the server is sometimes delayed it will be sound like normal

**Comment by WolverinDEV** [comment]
fixed
